src/Uno.UI/ts/WindowManager.ts

```diff
@@ -1389,37 +1388,15 @@ namespace Uno.UI {
 			}
 		}
 
-		private createUnconstrainedStyle(elementStyle: CSSStyleDeclaration, maxWidth: number, maxHeight: number): string {
-			const updatedStyles = <any>{};
+		private setUnconstrainedStyle(elementStyle: CSSStyleDeclaration, maxWidth: number, maxHeight: number): void {
 
-			for (let i = 0; i < elementStyle.length; i++) {
-				const key = elementStyle[i];
-				updatedStyles[key] = elementStyle.getPropertyValue(key);
-			}
-
-			if (updatedStyles.hasOwnProperty("width")) {
-				delete updatedStyles.width;
-			}
-			if (updatedStyles.hasOwnProperty("height")) {
-				delete updatedStyles.height;
-			}
+			elementStyle.removeProperty("width");
+			elementStyle.removeProperty("height");
 
 			// This is required for an unconstrained measure (otherwise the parents size is taken into account)
-			updatedStyles.position = "fixed";
-			updatedStyles["max-width"] = Number.isFinite(maxWidth) ? maxWidth + "px" : "none";
-			updatedStyles["max-height"] = Number.isFinite(maxHeight) ? maxHeight + "px" : "none";
-
-			let updatedStyleString = "";
-
-			for (let key in updatedStyles) {
-				if (updatedStyles.hasOwnProperty(key)) {
-					updatedStyleString += key + ": " + updatedStyles[key] + "; ";
-				}
-			}
-
-			// We use a string to prevent the browser to update the element between
-			// each style assignation. This way, the browser will update the element only once.
-			return updatedStyleString;
```

Comment:
This is unfortunately required, We'll have to find another way around this.

Reply:
@jeromelaban You mean, setting the style with cssText is different with setting it directly?

Reply:
The comment on that particular line is performance related, and this is the reason for the function complexity.